napari/layers/labels/labels.py

```diff
@@ -551,3 +614,18 @@ def paint(self, coord, new_label, refresh=True):
 
         if refresh is True:
             self.refresh()
+
+    def get_message(self):
+        msg = super().get_message()
+        # if this labels layer has properties
+        if self._label_index and self._properties:
+            # if the cursor is not outside the image or on the background
+            if self._value is not None:
+                if self._value in self._label_index:
+                    idx = self._label_index[self._value]
+                    for k, v in self._properties.items():
+                        if k != 'index':
+                            msg += f' {k}: {v[idx]}'
```

Comment:
@DragaDoncila playing with the example has convinced me that this should be:

```suggestion
                            msg += f', {k}: {v[idx]}'
```

Reply:
Yeah I did wonder. I also wondered what would happen if you had a long list of properties, are we happy to just print it all?